app.js

```diff
@@ -1,15 +1,25 @@
+const persist = require('choo-persist')
 const mount = require('choo/mount')
 const log = require('choo-log')
+const xtend = require('xtend')
 const choo = require('choo')
 
-const mainView = require('./pages/main')
+const opts = {
+  filter: (state) => {
+    state = xtend(state)
```

Comment:
I'd rather we don't get hung up on syntax, including making people learn things they don't otherwise know. Like every PR I make there's comments about syntax or new specs and I feel it slows us down - I don't wanna learn the quirks of classes or details of `Object.extend()` when we could be improving our design and fixing bugs instead.

I'm happy to have a convo about syntax and language features, but I don't feel it's enough to stall a PR over. Let's have that convo another time, perhaps in person when we hang out; think that'd work better than discuss it over text (: